packages/components/src/Text/Text.tsx

```diff
@@ -1,32 +1,37 @@
-import React, { ReactNode } from "react";
+import React, { PropsWithChildren } from "react";
 import { Typography, TypographyOptions } from "../Typography";
 
 interface TextProps {
   readonly variation?:
     | "default"
     | "subdued"
-    | "intro"
     | "success"
     | "error"
     | "warn"
     | "info";
-  readonly children: ReactNode;
-}
 
-export interface VariationMap {
-  [variation: string]: TypographyOptions;
+  readonly size?: "small" | "base" | "large";
 }
 
-export function Text({ variation = "default", children }: TextProps) {
-  const variationMap: VariationMap = {
-    default: { textColor: "greyBlueDark", size: "base" },
```

Comment:
Since `size` is no longer being defined for each variation, should we talk more in the Docz readme for this component about the interactions between size and variation? Or, provide some different examples that include some of these previous defaults?

Reply:
The only previous example that changed was `intro`. I added in a section about size that I think covers this. Do you have ideas for other variations to cover?

Reply:
No, actually, I think that covers it.